package.json

```diff
@@ -38,7 +38,7 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "express": "~4.16.4",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#task/lazyAttsNgsiv2-pr762",
```

Comment:
Revert before merging this PR, once telefonicaid/iotagent-node-lib#813 get merged

Reply:
Fixed in 5a7c710